tests/unit/util/test.inc

```diff
@@ -467,3 +467,99 @@ void run_mixing_GET_ARG_and_FOR_EACH(void)
 	zassert_equal(a[3], 4);
 	zassert_equal(a[4], 5);
 }
+
+static void test_PART_OF_ARRAY(void)
+{
+	size_t i;
+	size_t array[3];
+	uint8_t *const alias = (uint8_t *)array;
+
+	zassert_false(PART_OF_ARRAY(array, &array[-1]));
+	zassert_false(PART_OF_ARRAY(array, &array[ARRAY_SIZE(array)]);
```

Comment:
This is missing a closing ), but then local runs are also complaining for

```
/usr/local/google/home/fabiobaltieri/zephyrproject/zephyr/tests/unit/util/test.inc:534:9: error: implicit declaration of function ‘ztest_test_suite’; did you mean ‘ztest_test_skip’? [-Werror=implicit-function-declaration]
```

Does this need to be converted to the ztest api?